src/test/java/ua/com/rozetka/TestSuit01.java

```diff
@@ -122,20 +147,22 @@ public void Test04() throws AWTException, InterruptedException {
 
     @Test//Hover pointer on the PLP items
     public void Test05() throws InterruptedException, AWTException {
-        webDriver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
-        webDriver.get("https://rozetka.com.ua/");
-        webDriver.findElement(By.tagName("input")).sendKeys("ноутбу");
+        webDriver.get(url);
+        webDriver.manage().timeouts().implicitlyWait(3, TimeUnit.SECONDS);
+        webDriver.findElement(By.tagName("input")).sendKeys("ноутбу", ENTER);
+        //element.sendKeys(ENTER);
         Robot r = new Robot();
-        Actions action = new Actions(webDriver);
         r.keyPress(KeyEvent.VK_ENTER);
         webDriver.manage().window().maximize();
-        //r.keyPress(KeyEvent.VK_F11);
-        Thread.sleep(1000);
-        for (int i = 1; i < 6; i++) {
-        var we = webDriver.findElement(By.xpath("//section/rz-grid/ul/li["+i+"]"));
-        action.moveToElement(we).build().perform();
-            Thread.sleep(2000);
-        }
 
+        Actions action = new Actions(webDriver);
+
+
+        for (int i = 1; i < 5; i++) {
```

Comment:
what does "5" mean?

Reply:
It is only five times pointer hovers items